### PR TITLE
Plain SQL in 3.0

### PR DESCRIPTION
**Pull request by d6y**
This updates the example code for Chapter 6 (Plain SQL) for Slick 3.0 and adds the `tsql` example from the [Typechecking SQL blog post](http://underscore.io/blog/posts/2015/05/28/typechecking-sql.html).

# A note on branches 

- _2.1_ is a branch containing all the 2.1 stuff.
- _3.0_ is where the new stuff is happening
- _master_ is what we are publishing. Right now, it's the same as 2.1. When we publish the 3.0 EA book, we'll merge the 3.0 example code onto master.

I recommend we work by create feature branches from either _3.0_ or _2.1_ as appropriate.

_However_....  I screwed this up  :sob:  and I've pushed these changes directly into the _3.0_ branch. I intended to work on a feature branch for plain SQL, and then issue a PR.  Fixing it looks [too hard for small reward](http://stackoverflow.com/questions/1270514/undoing-a-git-push) so I retroactively created this branch only to create a PR to highlight the changes I made.

This PR can be merged as all it will do is bring the feature branch up to date with 3.0 :-/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,6 +95,7 @@ _joins.scala_ contains much that is commented out.  Remove the comments from aro
 
 This project contains two main examples:
```
Comment (jonoabroad) [inline README.md]: There are three examples below

File: chapter-06/build.sbt [hunk 2]
```diff
@@ -1,6 +1,6 @@
 name := "essential-slick-chapter-05"
```
Comment (jonoabroad) [inline chapter-06/build.sbt]: name is chapter-05 but directory is chapter-06. 

Which is correct?

File: chapter-06/src/main/scala/chapter06/chat_schema.scala [hunk 3]
```diff
@@ -1,24 +1,26 @@
 package chapter06
 
+import slick.driver.JdbcDriver
```
Comment (jonoabroad) [inline chapter-06/src/main/scala/chapter06/chat_schema.scala]: The upgrade guide indicates we should no longer use the JdbcDriver - http://slick.typesafe.com/doc/3.0.0/upgrade.html#jdbcdriver

Comment (d6y) [inline chapter-06/src/main/scala/chapter06/chat_schema.scala]: It looks like the _object_ `JdbcDriver` has been deprecated. I don't think we use that any place. The _trait_, we which use, looks to be still relevant.

**Pull request merged**